components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -1140,6 +1140,11 @@ def report_settings(module):
 # FEEDBACK_APP: 6 = OMERO.web
 FEEDBACK_APP = 6
 
+# For any given release of api, we may support
+# one or more versions of the api.
+# E.g. /api/v1.0/
+API_VERSIONS = ('0.1',)
```

Comment:
for the future would be handle to have that in the application package in main `__init__.py`